pkg/controllers/upload.go

```diff
@@ -60,7 +62,7 @@ func GetImgLogo(id int) []string {
 		return nil
 	})
 	if err != nil {
-		panic(err)
+		fmt.Println(err)
```

Comment:
What is the point of this? If error happens, it will simply output an error and continue working in this function. Please handle errors with the help of "logger" package.